openchrom/plugins/net.openchrom.chromatogram.xxd.report.supplier.csv/src/net/openchrom/chromatogram/xxd/report/supplier/csv/core/ConfigurableReport.java

```diff
@@ -44,13 +43,12 @@ public IProcessingInfo<File> report(File file, boolean append, List<IChromatogra
 					processingInfo.setProcessingResult(file);
 				} catch(IOException e) {
 					logger.warn(e);
-					processingInfo.addErrorMessage("OpenChrom CSV Chromatogram Report", "The report couldn't be created. An error occured.");
+					processingInfo.addErrorMessage("OpenChrom CSV Chromatogram Report", "The report couldn't be created. ", e);
```

Comment:
Instead of `CSV` it should be named `Excel`.

Reply:
No, this is still the CSV report. I just appended the stacktrace to the error message.